src/main/java/seedu/address/commons/util/DateTimeUtil.java

```diff
@@ -56,7 +70,12 @@ public static boolean isValidDateTime(String test) {
      * @return true if the test string is valid and false otherwise
      */
     public static boolean isValidTime(String test) {
-        return test.matches(TIME_VALIDATION_REGEX);
+        try {
```

Comment:
might wanna check if its null, javadocs specifies that it cannot be null, not sure if NPE will be thrown

Reply:
good point.